dts/bindings/misc/tenstorrent,bh-fwtable.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2025 Tenstorrent AI ULC
+# SPDX-License-Identifier: Apache-2.0
+
+description: Tenstorrent Blackhole Firmware Table
+
+compatible: "tenstorrent,bh-fwtable"
+
+include: base.yaml
+
+properties:
+
+  flash_dev:
+    type: phandle
+    required: true
+    description: |
+      Phandle to the flash device the firmware table resides in. If this property is
+      not specified, the firmware table will not be read from SPI flash.
```

Comment:
This is a TODO item - probably best handled in a follow-up. It means that the `flash_dev` property is not absolutely required, but we can revisit that after the PR is in.